modules/service/src/main/scala/lucuma/odb/graphql/input/ImagingScienceRequirementsInput.scala

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2016-2025 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.odb.graphql
+package input
+
+import cats.syntax.parallel.*
+import lucuma.core.enums.ScienceMode
+import lucuma.core.math.Angle
+import lucuma.odb.data.Nullable
+import lucuma.odb.graphql.binding.BooleanBinding
+import lucuma.odb.graphql.binding.Matcher
+import lucuma.odb.graphql.binding.ObjectFieldsBinding
+
+case class ImagingScienceRequirementsInput(
+  minimumFov:      Nullable[Angle],
+  narrowFilters:   Nullable[Boolean],
+  broadFilters:    Nullable[Boolean],
+  combinedFilters: Nullable[Boolean]
```

Comment:
If the mode is imaging, can these be `null`?

Reply:
If you cannot have `null` values when imaging, it seems like this should be `Option` rather than `Nullable`.  Same over in `SpectroscopyScienceRequirementsInput`. Or is the idea to assign it to `null` in order to go back to whatever the default was?